### PR TITLE
[PM-3226] Adding session to ReferenceEventRequest

### PR DESCRIPTION
**Pull request by aj-rosado**
## Type of change

```
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Adding the session value to reference data on trial-initiation

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Review by djsmith85 — APPROVED** [review]

**Review by JaredSnider-Bitwarden — APPROVED** [review]
LGTM!

File: apps/web/src/app/auth/trial-initiation/trial-initiation.component.ts [hunk 1]
```diff
@@ -94,6 +94,12 @@ export class TrialInitiationComponent implements OnInit, OnDestroy {
 
     if (this.referenceData.id === "") {
       this.referenceData.id = null;
+    } else {
+      const regex = /_ga_QBRN562QQQ=([^.]+)\.([^.]+)\.(\d+)/;
```
Comment (jlf0dev) [inline apps/web/src/app/auth/trial-initiation/trial-initiation.component.ts:98]: Can you please leave a comment explaining the intention of the regex?

**Review by jlf0dev — CHANGES_REQUESTED** [review]
Please leave a comment explaining the regex

**Review by jlf0dev — APPROVED** [review]
Thanks! LGTM

**Comment by aj-rosado** [comment]
Merging as requested by QA to test on cloud

**Pull request merged**